qa/integration-tests/src/test/java/io/camunda/zeebe/it/backup/BackupMultiPartitionTest.java

```diff
@@ -114,7 +114,8 @@ void createBackupStoreForTest() {
             S3.accessKey(),
             S3.secretKey(),
             Duration.ofSeconds(15),
-            true);
+            true,
+            false);
```

Comment:
:upside_down_face: Number of params to the constructur is increasing. A builder for the config might be better.